src/core/StelSkyCultureMgr.cpp

```diff
@@ -279,6 +282,25 @@ QString StelSkyCultureMgr::getCurrentSkyCultureHtmlLicense() const
 	return html;
 }
 
+QString StelSkyCultureMgr::getCurrentSkyCultureHtmlRegion() const
+{
+	QString html = "", region = currentSkyCulture.region.trimmed();
+	QString description = q_("The region is understood as the geographical area of origin of a given culture of the sky.");
+
+	// special case: modern sky culture
+	if (getCurrentSkyCultureID().contains("modern", Qt::CaseInsensitive))
+	{
+		// TRANSLATIONS: By the fact this is name of pseudo-region on Earth
+		region = N_("World");
+		description = q_("All modern sky cultures based on approved by IAU 88 constellations with standardized boundaries and used onto whole world. The origins all of these constellations are pan-European.");
```

Comment:
All modern sky cultures are based on the IAU-approved 88 constellations with standardized boundaries and are used worldwide. The origins of all these constellations are pan-European.